FinancialTracker/Firebase/DatabaseManager.swift

```diff
@@ -492,29 +508,149 @@ class DatabaseManager {
                     return
                 }
                 
-                if transferType == .send {
-                    self.addTransactionToCurrentUser(amount: amount, category: ExpenseCategory.transfer) { databaseError, _ in
+                guard let receiverCurrency = user.currency,
+                      let senderCurrency = currentUser.currency else {
+                    completionHandler(DatabaseError.unknown, nil)
```

Comment:
the `DatabaseError` is quite clear, it should not be unknown.

Reply:
There is no error type to be given to `DatabaseError.database()`

Reply:
same as above

Reply:
Same as above :)